libs/src/index.js

```diff
@@ -67,7 +68,7 @@ class AudiusLibs {
       useExternalWeb3: true,
       externalWeb3Config: {
         web3: web3Instance,
-        ownerWallet: walletOverride || wallets[0]
+        ownerWallet: walletOverride || wallets[walletIndex]
```

Comment:
let's remove this - it's already in cnode but this walltes + index paradigm is super not reflective of prod - just passin gin the right wallet should be sufficient

Reply:
done as discussed